tests/foreman/ui/test_discoveryrule.py

```diff
@@ -83,6 +83,11 @@ def reader_user(module_location, module_org):
     return reader_user
 
 
+def gen_int32(min_value=1):
```

Comment:
Consider contributing to fauxfactory to define this, since its based off of `gen_integer`.